docs/source/tutorials/private_network.md

```diff
@@ -1,70 +1,44 @@
 ## Start private Kagome network
 
-In this tutorial we will learn how to start a blockchain nework with a validator and block producing nodes
+In this tutorial we will learn how to start a blockchain network with a validator and block producing nodes
 
 First go to tutorial's folder:
 
 ```bash
 cd examples/network
 ```
 
-### Execute validating node
+### Execute first validating node
 
-First we execute validating node in the simillar way we did it during previous tutorial. This node will produce and finalize blocks.
+First we execute validating node in the similar way we did it during previous tutorial. This node will produce and finalize blocks.
 
 To start with let's navigate into the node's folder:
 
 ```bash
 kagome_validating \
     --genesis testchain.json \
     --keystore testkeystore1.json \
-     --leveldb test_ldb1 \ 
-     --p2p_port 11122 \
-     --rpc_http_port 11133 \
-     --rpc_ws_port 11144 \
-     -f
+    -l test_ldb1 --p2p_port 11122 \
+    --rpc_http_port 11133 \
+    --rpc_ws_port 11144 \
+    -f \
```

Comment:
I think it would be better to use long-name cli-args coherently like in another places